flang/lib/Evaluate/intrinsics.cpp

```diff
@@ -921,6 +921,11 @@ static const IntrinsicInterface genericIntrinsicFunction[]{
             {"back", AnyLogical, Rank::elemental, Optionality::optional},
             DefaultingKIND},
         KINDInt},
+    {"secnds",
+        {{"x", TypePattern{RealType, KindCode::exactKind, 4}, Rank::scalar,
```

Comment:
Are you sure that the reference argument is always `REAL(4)`, or is it the default `REAL` kind?  Are the semantics in GNU Fortran affected when the default `REAL` kind is changed on the command line?

`Optionality::required` and `Intent::In` are the default settings, so you don't need to specify them again here.